### DISCUSSION TITLE
Anyone using Stereo_Tool?

### DISCUSSION THREAD

**Discussion by bombelman** [AzuraCast/AzuraCast#7295] [General]
Anyone using Stereo_Tool with AzuraCast?
I would like to add more low end, and more airy richness to my audio..
What is your experience with using Stereo_Tool ? Does it enhance your listening experience?

Can I also achieve a similar result with Master_me?

**Comment by Heimatradio** [comment]
real, beter Sound with StereoTool, find out your settings for ST
this one is running with ST https://stream.stream-online.eu:9000/MP3-320

**Comment by bombelman** [comment]
awesome😍 which licenses are you using if I may ask? @Heimatradio

**Reply by Heimatradio** [reply]
Basic licence from StereoTool

**Comment by BornKillaz** [comment]
I'm using **master_me**, **lsp_graph_equalizer_x16_stereo** and **lsp_mb_expander_stereo**. 
In my opinion, not as nice as ST, but it sounds good and... it's free https://basstik.com

**Reply by bombelman** [reply]
the way you have it set up, can you interact with the gui also?

**Reply by BornKillaz** [reply]
Nop. Just plain text configuration in liquidsoap. This [topic](https://github.com/AzuraCast/AzuraCast/discussions/6191#discussioncomment-8851249) was a great resource for me.

**Comment by JeffersonStarseed** [comment]
I like both of the tools.  Master_Me is a no-brainer for those that want to use Azuracast's (now) built-in pre-set audio-processing tool for the front-end before streaming out to the listener, it will slightly compress, smooth-out and brighten the sound.

However, Stereo Tool is going to improve the sound a whole lot more, especially if your looking for that extra low-end and percussive emphasis (without the spikes).  The "only" thing I "don't" like about Stereo Tool is that most of the presets available to use seem to be compression-heavy to me, and you can sometimes hear (or feel) compression and auto-dynamics "kicking in-and-out" in the music.  Because of this, I recommend making sure that your audio input level or music file volume going into Stereo Tool is not set higher than it should be, so the compression and auto-dynamics do not overwork.

The controls/settings for Stereo Tool are massive.  It's not for the faint of heart, but there are loads of possibilities and options available to change or tweak.  You have to know what you're doing when you do manual editing, and you have to learn it and take your time with it.  I'm still learning.  I invested in ST 'Basic' along with the 'Declipper' and 'Advanced Dynamics' packages.  Running the 'Declipper' portion of Stereo Tool live in the stream is amazing and it really cleans-up some of the distortion and restores some of the fidelity from the original music files.  For myself, I tried taking some of the presets that Stereo Tool came with and starting tweaking those manually until I got what I wanted the audio to sound like.  I did the experimentation and manual preset-editing with Stereo Tool installed on my computer, and once I was happy with my new preset, I installed both Stereo Tool and later my custom "Preset" into Azuracast, quick and easy.  **_When it's set properly_**, Stereo Tool can make your online station sound like a good professional FM radio station.

Stereo Tool just went to version 10.30 in April (2024) which has a lot of major improvements, and even more things improved in their Beta (10.31).  At the same time, Azuracast keeps making improvements to itself, but the ability to install both Stereo Tool via the admin page, ...and your own custom ST preset config file 'per station' ...directly from within the Azuracast platform GUI/dashboard is huge.
